apps/web/src/app/(dashboard)/admin/page.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Archive,
+  File,
+  FileX2,
+  LucideIcon,
+  User as LucideUser,
+  Mail,
+  MailOpen,
+  PenTool,
+  Send,
+  UserPlus2,
+  UserSquare2,
+} from 'lucide-react';
+
+import { getDocsCount } from '@documenso/lib/server-only/admin/get-documents';
+import { getRecipientsStats } from '@documenso/lib/server-only/admin/get-recipients';
+import {
+  getUsersCount,
+  getUsersWithSubscriptionsCount,
+} from '@documenso/lib/server-only/admin/get-users';
+import {
+  ReadStatus as InternalReadStatus,
+  SendStatus as InternalSendStatus,
+  SigningStatus as InternalSigningStatus,
+} from '@documenso/prisma/client';
+
+import { CardMetric } from '~/components/(dashboard)/metric-card/metric-card';
+
+type TCardData = {
+  icon: LucideIcon;
+  title: string;
+  status:
+    | 'TOTAL_RECIPIENTS'
+    | 'OPENED'
+    | 'NOT_OPENED'
+    | 'SIGNED'
+    | 'NOT_SIGNED'
+    | 'SENT'
+    | 'NOT_SENT';
+}[];
+
+const CARD_DATA: TCardData = [
```

Comment:
For this I'd consider dropping `[]` from `TCardData` declaration and instead adding it to the constant declaration.

e.g. `CARD_DATA: TCardData[] = [...]`